exotica_python/CMakeLists.txt

```diff
@@ -16,4 +16,6 @@ include_directories(
 )
 
 pybind_add_module(pyexotica MODULE src/Exotica.cpp)
-install(TARGETS pyexotica LIBRARY DESTINATION ${CATKIN_PACKAGE_PYTHON_DESTINATION})
+install(TARGETS pyexotica LIBRARY DESTINATION ${CATKIN_GLOBAL_PYTHON_DESTINATION})
+catkin_install_python(PROGRAMS scripts/example_aico.py scripts/example_ompl.py scripts/example_ik.py scripts/publish_trajectory.py DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
```

Comment:
Is there a simple way to include it if we put it somewhere else?
We may have to set up that subdirectory in the global path and add both the ```*.so``` and ```publish_trajectory.py``` into ```_init__.py```.